func/domain.sh

```diff
@@ -481,7 +481,7 @@ update_domain_zone() {
                                             7200
                                             3600
                                             1209600
-                                            180 )
+                                            3600 )
```

Comment:
180 mean it will be updated every 5 min instead of every 1 hour. We have discussed it in the past and we don't see any reason to change it.

Reply:
I did not intend to submit this again. I already stated the reason for this change. I was just doing two other changes. Don't know why this came together